src/modules/intelligence/application/event_handlers/analysis_completed_handler.py

```
@@ -0,0 +1,63 @@
+                if group_result["failed"] > 0:
+                    logger.warning("auto_group_translation_partial", language=lang, failed=group_result["failed"])
+            except Exception as e:
+                logger.error("auto_group_translation_error", language=lang, error=str(e))
+                self._try_rollback()
+
+    def _try_rollback(self) -> None:
+        if self._session_rollback:
+            try:
+                self._session_rollback()
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)